bedrock/firefox/templates/firefox/developer/firstrun.html

```diff
@@ -24,6 +24,8 @@
 {% block body_id %}firefox-developer{% endblock %}
 {% block body_class %}firefox-developer firefox-developer-firstrun{% endblock %}
 
+{% block site_header %}{% endblock %}
```

Comment:
This now makes aurora whatsnew and firstrun aligned with release/beta and nightly in a way that it doesn't show the main site header and navigation (wnp implicitly from the base diff here, firstrun explicitly on this line).